.github/workflows/deploy-xtgeo-pypi.yml

```diff
@@ -4,83 +4,164 @@ on:
 
       - name: Install cibuildwheel
-        run: |
+        run: >
+          python -m pip install pip -U &&
+          python -m pip install wheel &&
           python -m pip install cibuildwheel
 
-      - name: Build wheels and test
+      - name: Build wheels and test python < 3.9
+        if: matrix.cibw_python != 'cp39'
         run: |
           python -m cibuildwheel --output-dir wheelhouse
 
-      - name: Check wheels and paths (ubuntu only)
+      - name: Build wheels and test python 3.9
+        if: matrix.cibw_python == 'cp39'
+        # note tests are a bit restricted due to missing pytables wheel
+        env:
+          CIBW_TEST_COMMAND: >
+            pushd {project} &&
+            pytest tests --disable-warnings --ignore tests/test_well
+
         run: |
-          ls -l wheelhouse
-          ls -latr
-        if: matrix.os == 'ubuntu-latest'
+          python -m cibuildwheel --output-dir wheelhouse
 
-      - name: Publish to PYPI
+      - name: Publish to PYPI MacOS
         env:
           TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
           TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+        run: >
+          pip install twine &&
+          twine upload wheelhouse/*
+
+  deploy_windows_cibuildwheel:
+    name: CIBW deploy ${{ matrix.cibw_python }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+
+    env:
+      CIBW_BEFORE_ALL: choco install -y --no-progress --allow-empty-checksums -r swig --version=4.0.1"
+      CIBW_BEFORE_TEST: >
+        pushd {project} &&
+        pip install -r requirements/requirements.txt &&
+        pip install -r requirements/requirements_test.txt &&
+        git clone --depth 1 https://github.com/equinor/xtgeo-testdata ../xtgeo-testdata
+
+      # a test with forks is skipped as this calls python in a subprocess, where
+      # the cibuildwheel on windows cannot find the xtgeo module
+      CIBW_TEST_COMMAND: >
+        pushd {project} &&
+        pytest tests --disable-warnings --ignore-glob="*forks.py"
+      CIBW_BUILD: ${{ matrix.cibw_python }}-win_amd64
+
+    strategy:
+      matrix:
+        os: [windows-latest]
+        cibw_python: [cp36, cp37, cp38, cp39]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+
+      - name: Install cibuildwheel
+        run: >
+          python -m pip install pip -U &&
+          python -m pip install wheel &&
+          python -m pip install cibuildwheel
+
+      - name: Build wheels and test python < 3.9
+        if: matrix.cibw_python != 'cp39'
+        run: |
+          python -m cibuildwheel --output-dir wheelhouse
+
+      - name: Build wheels and test python 3.9
+        if: matrix.cibw_python == 'cp39'
+        # note tests are a bit restricted due to missing pytables wheel
+        env:
+          CIBW_TEST_COMMAND: >
+            pushd {project} &&
+            pytest tests --disable-warnings --ignore tests/test_well
+            --ignore-glob="*forks.py" -x
```

Comment:
Splitting this on multiple lines works, right?

Reply:
Yes, in yaml:
```
command: > 
hello
world
```

is parsed as `"hello world"`